hrm-service/README.md

```diff
@@ -1,42 +0,0 @@
-## Running tests
```

Comment:
This was (mostly) duplicated in the root level readme so I centralized it there.